db/internal_stats.cc

```diff
@@ -482,6 +486,9 @@ const std::unordered_map<std::string, DBPropertyInfo>
         {DB::Properties::kLiveSstFilesSize,
          {false, nullptr, &InternalStats::HandleLiveSstFilesSize, nullptr,
           nullptr}},
+        {DB::Properties::kLiveSstFilesSizeAtTemperature,
+         {true, &InternalStats::HandleLiveSstFilesSizeAtTemperature, nullptr,
```

Comment:
Is there a reason it is returning string, rather than integer? It feels that it always returns integers.

Reply:
Int one doesn't support suffix, so similar to [`Properties::kNumFilesAtLevelPrefix`](https://github.com/facebook/rocksdb/blob/a63ee24a191b10d9fd615002eb3dea35345aceb0/db/internal_stats.cc#L379) it returns a string.